cmd/controller-manager/app/options/options_test.go

```diff
@@ -17,11 +17,11 @@ limitations under the License.
 package options
 
 import (
-	"reflect"
 	"testing"
 	"time"
 
 	"github.com/spf13/pflag"
+	equality "k8s.io/apimachinery/pkg/api/equality"
```

Comment:
1. pkg name is already `equality` so there is no need to add a alias: )

Reply:
though I removed this everywhere, will clean this soon

Reply:
updated